src/modules/control/test/README.md

```diff
@@ -17,7 +17,11 @@ The unittest in control package is written with gtest, Google's C++ unittest fra
 
 ### To Build and Run the test:  
 * ```catkin_make tests```
-* ```roscore && rosrun control control_test```   
+* For running the control lateral test:  
```

Comment:
Is there any way we can run multiple tests at once in some launch file (similar to what we have in [the geodesy tests](https://github.com/SJSU-AD/FusionAD/blob/test_deployment/src/modules/geodesy/test/test_geodesy.test)) so this doesn't just turn into a long running list of commands that takes excessive manual effort to run?

Reply:
I believe that `catkin_make run_tests` is able to run all of the unit tests that are declared in each respective `CMakeLists.txt` file in the repo.

Reply:
```catkin_make``` only builds all the tests, I will create a launch file to run all these tests.

Reply:
Try out `catkin_make run_tests` before you make the launch file, it builds all of the tests and then runs them one by one.

Reply:
Actually disregard that, you should build the launch file because the robot_localization tests failed to build

Reply:
I do not recommend using ```run_tests``` as it would build the entire AD workspace's test at once. Seems like the tests in loam is not quite building, which will throw an error when you build run_test and remove all the test results for all other packages' building tests. 

Instead, we will settle by using an individual target run_test command ```catkin_make run_tests_control```, which only builds and run control tests. For other packages, the command would be ```catkin_make run_tests_$PACAKGENAME```.